src/uu/cut/src/cut.rs

```diff
@@ -341,10 +353,10 @@ fn cut_files(mut filenames: Vec<String>, mode: Mode) -> i32 {
                 continue;
             }
 
-            exit_code |= match mode {
-                Mode::Bytes(ref ranges, ref opts) => cut_bytes(stdin(), ranges, opts),
-                Mode::Characters(ref ranges, ref opts) => cut_bytes(stdin(), ranges, opts),
-                Mode::Fields(ref ranges, ref opts) => cut_fields(stdin(), ranges, opts),
+            match mode {
+                Mode::Bytes(ref ranges, ref opts) => cut_bytes(stdin(), ranges, opts)?,
+                Mode::Characters(ref ranges, ref opts) => cut_bytes(stdin(), ranges, opts)?,
+                Mode::Fields(ref ranges, ref opts) => cut_fields(stdin(), ranges, opts)?,
```

Comment:
The suggestion in the other comment applies here too

Reply:
Done!